### ISSUE TITLE
Computer terminal art?

### ISSUE THREAD

**Issue by Kelvin-Lee** [Kelvin-Lee/kelvin-lee.github.io#4]
A nice design enhancement would be to encase the pixel art within something that looks like a computer screen.

Probably depends on #2 to be finished first, since part of solving #2 requires choosing a fixed pixel grid size.

**Comment by Kelvin-Lee** [comment]
No longer relevant; old design.
